compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1261,6 +1273,39 @@ LogicalResult AttentionOp::verify() {
     return failure();
   }
 
+  // Additional check case if mask exists
+  if (getMaskMap().has_value() &&
+      failed(checkShape("Mask", getMaskType()->getShape(),
+                        getMaskMap().value()))) {
```

Comment:
```suggestion
  // Additional check case if mask exists.
  if (auto maskMap = getMaskMap()) {
    if (failed(checkShape("Mask", getMaskType()->getShape(), *maskMap)) {
```

This version should be easier to debug because you can get the maskMap value without calling helper functions.

Reply:
big +1

Reply:
Gotcha